### PR TITLE
adding statusMessage

### PR DESCRIPTION
**Pull request by KurtStam**
Added the Integration.statusMessage back, and added a message for the Timed integration example. 

I also increased the db version so it will refresh the data on staging when merged and deployed.

### CODE REVIEW

**Comment by kahboom** [comment]
I may not be a reviewer, but I strongly approve of this.

File: dao/src/main/resources/com/redhat/ipaas/dao/deployment.json [hunk 1]
```diff
@@ -498,7 +498,8 @@
       "name": "Timed Pull to Post Example",
       "description": "This is an example of a Timed Pull to Post Integration",
       "configuration": "",
-      "desiredStatus": "Activated",
+      "desiredStatus": "Deactivated",
```
Comment (kahboom) [inline dao/src/main/resources/com/redhat/ipaas/dao/deployment.json]: Yes! Now we can use `desiredStatus` as well.

Comment (kahboom) [inline dao/src/main/resources/com/redhat/ipaas/dao/deployment.json]: Which was available, but there was no point in using as none of the data used it. @gashcrumb WDYT?

Comment (gashcrumb) [inline dao/src/main/resources/com/redhat/ipaas/dao/deployment.json]: @kahboom we've actually setting `desiredStatus` when saving the integration depending on which button you use.  Recent change I made gets this state reflected in the UI though really the state that we should be showing is actually `currentStatus`, not `desiredStatus`.

**Comment by KurtStam** [comment]
merge please

**Pull request merged**